File: languages/ruby/fizzbuzz.rb
(1..100).each do |i|
  line =
    if i % 15 == 0 then 'FizzBuzz'
    elsif i % 3 == 0 then 'Fizz'
    elsif i % 5 == 0 then 'Buzz'
    else i
    end
  puts line
end

File: languages/elixir/fizzbuzz.exs
Enum.each(1..100, fn i ->
  IO.puts(
    cond do
      rem(i, 15) == 0 -> "FizzBuzz"
      rem(i, 3) == 0 -> "Fizz"
      rem(i, 5) == 0 -> "Buzz"
      true -> i
    end
  )
end)
